__tests__/integration/configuration/privnet.yml

```diff
@@ -0,0 +1,32 @@
+# This file is the Private Network Configuration for the Fullnode
+# It is consumed by the docker-compose.yml file on the integration folder.
+# For more information, refer to:
+# https://github.com/HathorNetwork/rfcs/blob/master/text/0033-private-network-guide.md
+
+# This genesis adds the funds to the following wallet seed:
+# avocado spot town typical traffic vault danger century property shallow divorce festival spend attack anchor afford rotate green audit adjust fade wagon depart level
+
+P2PKH_VERSION_BYTE: x49
+MULTISIG_VERSION_BYTE: x87
+NETWORK_NAME: privatenet
+BOOTSTRAP_DNS: []
+ENABLE_PEER_WHITELIST: false
+
+# Genesis stuff
+GENESIS_OUTPUT_SCRIPT: 76a91466665b27f7dbc4c8c089d2f686c170c74d66f0b588ac
+GENESIS_BLOCK_TIMESTAMP: 1643902665
+GENESIS_BLOCK_NONCE: 4784939
+GENESIS_BLOCK_HASH: 00000334a21fbb58b4db8d7ff282d018e03e2977abd3004cf378fb1d677c3967
+GENESIS_TX1_NONCE: 0
+GENESIS_TX1_HASH: 54165cef1fd4cf2240d702b8383c307c822c16ca407f78014bdefa189a7571c2
+GENESIS_TX2_NONCE: 0
+GENESIS_TX2_HASH: 039906854ce6309b3180945f2a23deb9edff369753f7082e19053f5ac11bfbae
+
+MIN_TX_WEIGHT_K: 0
+MIN_TX_WEIGHT_COEFFICIENT: 0
+MIN_TX_WEIGHT: 1
+REWARD_SPEND_MIN_BLOCKS: 1
+
+CHECKPOINTS: []
```

Comment:
This was not on the old config, is it a new required attribute?

Reply:
It's not required, since we extends the mainnet yml, but I wanted to make sure this won't break anything when the sync v2 becomes the default sync protocl